frontend/database/172_add_certificate_fields.sql

```diff
@@ -0,0 +1,11 @@
+-- Courses
+ALTER TABLE `Courses`
+  ADD COLUMN `minimum_progress_for_certificate` int(11) DEFAULT NULL
+    COMMENT 'Progreso mínimo que debe cumplir el estudiante para que se le otorgue el diploma del curso. NULL indica que el curso no da diplomas.';
+
+-- Contests
+ALTER TABLE `Contests`
+  ADD COLUMN `certificate_cutoff` int(11) DEFAULT NULL
+    COMMENT 'Número de concursantes a premiar con diplomas que mencionan su lugar en el ranking',
+  ADD COLUMN `certificates_status` enum('uninitiated', 'queued', 'generated', 'retryable_error', 'fatal_error')
```

Comment:
```suggestion
  ADD COLUMN `certificates_status` enum('uninitiated', 'queued', 'generated', 'retryable_error', 'fatal_error') NOT NULL
```

welp, se nos fue esto en la revisión pasada. puedes agregarle `NOT NULL` para que no permita que sea nulo aquí ni en ninguna de las anotaciones de Psalm en los PHP?